about.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: About
+permalink: /about/
+---
+
+This reference is currently a work in progress. Not all pages may be complete and pages may be updated at any time without notice.
```

Comment:
> pages may be updated at any time without notice

Should we show people how to see changes for a page? Or better yet, have a link on the bottom of each page to the history page on GitHub? (e.g. https://github.com/payments-reference/payments-reference.github.io/commits/master/glossary.md)